azure-pipelines.yml

```diff
@@ -28,15 +28,6 @@ jobs:
   - script: sudo sysctl -w net.inet.udp.maxdgram=65535
   - script: sudo -E bash scripts/test.sh
 
-- job: windows_2016_vs2017_winpcap
-  pool:
-    vmImage: 'VS2017-Win2016'
-  steps:
-  - script: choco install -y winpcap
-  - script: dotnet restore -s https://api.nuget.org/v3/index.json
-  # Disable WinDivert to avoid an internal CLR error due to secure boot having been enabled
-  - script: bash scripts/test.sh --filter TestCategory!=WinDivert
-
```

Comment:
@chmorgan It's true that WinPcap is deprecated, but it's the only way to test rpcap on Windows, so we will need to keep at least one of the CI jobs.